src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -217,6 +206,16 @@ func runClientProxy(cfg *config.Config) {
 		exit(err)
 	}
 
+	if *clearProxySettings {
+		// This is a workaround that attempts to fix a Windows-only problem where
+		// Lantern was unable to clean the system's proxy settings before logging
+		// off.
+		//
+		// See: https://github.com/getlantern/lantern/issues/2776
+		doPACOff("http://" + cfg.UIAddr + "/proxy_on.pac")
```

Comment:
Not a big deal, but we seem to use the ``fmt.Sprintf("http://%s/proxy_on.pac", cfg.UIAddr)`` convention elsewhere, and it's cleaner to me.